scripts/wrappers/common/utils.py

```diff
@@ -247,15 +247,28 @@ def is_community_addon(arch, addon_name):
 
 def get_available_addons(arch):
     available = []
+    strict = is_strict()
     for dir in os.listdir(snap_common() / "addons"):
         try:
             addons_yaml = snap_common() / "addons" / dir / "addons.yaml"
             with open(addons_yaml, "r") as fin:
                 addons = yaml.safe_load(fin)
 
             for addon in addons["microk8s-addons"]["addons"]:
-                if arch in addon["supported_architectures"]:
+                to_be_added = True
```

Comment:
Since the `to_be_added` variable is initially true and becomes false as needed, isn't it better to simply have a `continue` statement when any of the predicates below are false?